integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataLoadRDD.scala

```diff
@@ -462,3 +478,133 @@ class CarbonDataLoadRDD[K, V](
   }
 }
 
+class CarbonRDDDataLoadRDD[K, V](
```

Comment:
Please add needful annotate for new class CarbonRDDDataLoadRDD. (it is difficult to know what is the different of CarbonRDDDataLoadRDD, CarbonDataLoadRDD)
Same comments for other place of this PR :  please provide needful annotate for new classes and new functions.